src/app/project/project.component.ts

```diff
@@ -269,4 +273,30 @@ export class ProjectComponent implements OnInit {
       return 0;
     }
   }
+
+  getProjectGeoAccession() {
+    if (this.project?.content['geo_series_accessions'])
```

Comment:
This if statement is unnecessary if you use the ? Syntax

Reply:
You can usee the tsignore annotation in that case or provide the type of content inline

Reply:
what do you mean by "type of content inline"?

Reply:
also, I'd personally rather have a longer conditional check than use the tsignore annotation

Reply:
Add the type of the `content` field inline e.g. `let obj: { property: string; } = { property: "foo" };`

Reply:
see no problem with above syntax. check is on `content['geo_series_accessions']` being present. Not sure if a check on length before return index 0 is needed though.

Reply:
@prabh-t that check is in case the value against `geo_series_accession` isn't defined

Reply:
is this any better?

```
const content: { geo_series_accessions?: string[] } = this.project?.content;
const geoAccessions = content?.geo_series_accessions;
return geoAccessions && geoAccessions.length > 0 ? geoAccessions[0] : undefined;
```

and yes @prabh-t we need to add the length check, empty array is truthy.

Reply:
ah this works:
```
const content: { geo_series_accessions?: string[] } = this.project?.content;
return content?.geo_series_accessions?.[0];
```